keras/utils/torch_utils_test.py

```diff
@@ -53,4 +69,50 @@ def test_module_autowrapping(self):
         self.assertEqual(len(model.fc2.trainable_weights), 2)
         self.assertEqual(len(model.trainable_weights), 4)
         model.compile(optimizer="sgd", loss="mse")
-        model.fit(np.random.random((3, 2)), np.random.random((3, 4)))
+        model.fit(np.random.random((3, 2)), np.random.random((3, 2)))
+
+    @parameterized.parameters(
+        {'use_batch_norm': False},
+        {'use_batch_norm': True},
+    )
+    def test_load_weights(self, use_batch_norm):
+        # Test loading weights
+        temp_filepath = os.path.join(self.get_temp_dir(), "mymodel.weights.h5")
+        model = Classifier(use_batch_norm)
+        model.compile(optimizer="sgd", loss="mse")
+        x, y = np.random.random((3, 2)), np.random.random((3, 1))
+        x_test, y_test = np.random.random((3, 2)), np.random.random((3, 1))
+        ref_loss1 = model.evaluate(x_test, y_test)
+        model.fit(x, y)
+        ref_loss = model.evaluate(x_test, y_test)
+        model.save_weights(temp_filepath)
+
+        new_model = Classifier(use_batch_norm)
+        new_model(np.random.random((3, 2)))
+        new_model.compile(optimizer="sgd", loss="mse")
+        new_model.load_weights(temp_filepath)
+        for ref_w, new_w in zip(model.get_weights(), new_model.get_weights()):
+            self.assertAllClose(ref_w, new_w, atol=1e-5)
+        loss = new_model.evaluate(x_test, y_test)
+        self.assertAllClose(ref_loss, loss, atol=1e-5)
+
+    @parameterized.parameters(
+        {'use_batch_norm': False},
+        {'use_batch_norm': True},
+    )
+    def test_serialize_model(self, use_batch_norm):
+        # Test loading saved model
+        temp_filepath = os.path.join(self.get_temp_dir(), "mymodel.keras")
+        model = Classifier(use_batch_norm)
+        model.compile(optimizer="sgd", loss="mse")
+        x, y = np.random.random((3, 2)), np.random.random((3, 1))
+        x_test, y_test = np.random.random((3, 2)), np.random.random((3, 1))
+        model.fit(x, y)
+        ref_loss = model.evaluate(x_test, y_test)
+        model.save(temp_filepath)
+
+        new_model = saving.load_model(temp_filepath)
```

Comment:
I don't understand how this could possibly work in the general case if we don't implement `from_config`/etc.

Reply:
Added `get_config` to `Classifier` in unit test.